mne_connectivity/base.py

```diff
@@ -221,6 +290,18 @@ def shape(self):
         """Shape of raveled connectivity."""
         return self.xarray.shape
 
+    @property
+    def n_nodes(self):
+        return self.attrs['n_nodes']
+
+    @property
+    def method(self):
+        return self.attrs['method']
+
+    @property
+    def indices(self):
```

Comment:
These all need some minimal docstring (can be a single line with a human-readable description)

Reply:
Done!